tools/clang/unittests/HLSLExec/LongVectors.cpp

```diff
@@ -569,6 +569,23 @@ void runAndVerify(ID3D12Device *D3DDevice, bool VerboseLogging,
                                 ValidationConfig.Type, VerboseLogging));
 }
 
+template <typename IN_TYPE, typename OUT_TYPE>
+static std::vector<OUT_TYPE> TransformVector_1(
+    const InputSets<IN_TYPE> &Inputs,
+    function<OUT_TYPE(const IN_TYPE &, OUT_TYPE &)> TransformFunc) {
```

Comment:
Ugh - somewhere there's a `using namespace std` that shouldn't be there.  This should be qualified with `std::function` - although I'm going to ask that you don't use `std::function` so this is somewhat moot.